vue-app/src/components/Links.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <a
+    v-if="isExternal && hideArrow"
+    :class="className"
+    :href="to"
+    :aria-label="ariaLabel"
+    target="_blank"
+    rel="noopener"
+  >
+    <slot />
+  </a>
+  <a
+    v-else-if="isExternal"
+    :class="`external-link ${className}`"
+    :href="to"
+    :aria-label="ariaLabel"
+    target="_blank"
+    rel="noopener"
+  >
+    <slot />
+  </a>
```

Comment:
Using object syntax to define dynamic classes https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax I think we can make this simpler.
```js
<a
  v-if="isExternal"
  :class="{ 'external-link': !hideArrow, [className]: !!className }"
>
  <slot />
</a>
```

Reply:
[pending thought]
I'm wondering whether there is a way in Vue to avoid using a different name (`className`) to pass a custom class. I would ideally use `class` from the consumer.
```js
// right now
<links :className="....">

// what I would really like
<links :class="....">
```

Reply:
More I think about it, do we even need to pass a prop for this at all? Couldn't we just assign a class to <links> wherever it is used and customize styling as needed in the parent component?

ie. `BackToProjects.vue`
We assign classes to the <links> component that is embedded. The CSS is declared within this file, and is used to custom style the resulting link, without passing 'className' prop

Testing locally, I tried overriding classes for external links using this method as well and seems to be working. I think this prop can be removed entirely

Reply:
Ha! read this https://vuejs.org/v2/guide/components-props.html#Replacing-Merging-with-Existing-Attributes
It will merge them (only for `class` and `style` attrs), cool!

So, yes, we are good if you remove that `className`. Any attr defined from the consumer perspective will be automatically spread in the root element of the component.